contest/test_clarification.py

```diff
@@ -0,0 +1,246 @@
+from django.test import TestCase, Client
+from django.core.urlresolvers import reverse
+
+from datetime import datetime, timedelta
+
+from contest.models import Contest, Clarification
+from utils.test_helper import *
+
+
+
+class Tester_Contest_ask(TestCase):
+    """ test view 'contest:ask' """
+
+    def setUp(self):
+        create_test_admin_user()
+        # create 6 judge level users
+        # (1 contest owner, 2 coowners,
+        #  2 problem owners, 1 contestant)
+        create_test_judge_user(6)
+        # create 51 normal users (50 contestants)
+        create_test_normal_user(51)
+        self.ADMIN_USER = get_test_admin_user()
+        self.ADMIN_CLIENT = get_test_admin_client()
+        self.JUDGE_USERS = [get_test_judge_user(i) for i in range(6)]
```

Comment:
At that time, I was not sure whether it would failed when running unit tests if we extend `TestCase` class.
But actually, it works!
I've collected the common part and extended TestCase in new commits.